django/cantusdb_project/main_app/views/chant.py

```diff
@@ -1389,9 +1389,17 @@ def get_context_data(self, **kwargs):
 
         # 4064 is the id for the sequence database
         if source.segment.id == 4064:
-            queryset = source.sequence_set.order_by("folio", "s_sequence")
+            queryset = source.sequence_set.annotate(
+                is_sequence=Value(True)
```

Comment:
Why not a single variable, `record_type='chant' | 'sequence'`.

(I'm a big proponent of data modelling where you "make impossible states impossible". In this case, you would never have `is_chant = True` and `is_sequence = True` since that would be an error. So model your data to make this state impossible!)

If you're interested: 

https://www.youtube.com/watch?v=IcgmSRJHu_8

https://kentcdodds.com/blog/make-impossible-states-impossible

Reply:
I like this suggestion - thanks. Let me make this change.